README.md

```diff
@@ -36,7 +36,7 @@ Add the following to your build.gradle:
 	        }
 	    }
 	    dependencies {
-	      classpath "gradle.plugin.com.greensopinion.gradle-android-eclipse:android-eclipse:1.1"
+	      classpath "gradle.plugin.com.greensopinion.gradle-android-eclipse:gradle-android-eclipse:1.1"
```

Comment:
Good catch!  Not sure how I went wrong here.  Your suggested change is aligned with the reference docs: https://plugins.gradle.org/plugin/com.greensopinion.gradle-android-eclipse

Reply:
Yes, that's where I found the right way... I tried the line from your blog, and the one in the README, that's how I caught it :-)